src/components/accordion-tab/accordion-tab.scss

```diff
@@ -0,0 +1,28 @@
+:host {
+  display: block;
+}
+
+details {
+  font-family: 'Open Sans', sans-serif;
+  width: 620px;
+}
+
+details > summary {
```

Comment:
As discussed: wrong selector here. Please use the add the summary & p styling directly in the details.